app/code/Magento/Store/Model/ResourceModel/StoreWebsiteRelation.php

```diff
@@ -41,4 +43,29 @@ public function getStoreByWebsiteId($websiteId)
         $data = $connection->fetchCol($storeSelect);
         return $data;
     }
+
+    /**
+     * Get website store data
+     *
+     * @param string $websiteId
+     * @param bool $available
+     * @return array
+     */
+    public function getWebsiteStores(string $websiteId, bool $available = false): array
```

Comment:
Why `$websiteId` has strict type `string`?
Generally, it represented via `int` strict types in the system.
My suggestion is to avoid ambiguous approach to do not 'surprise' anyone.
```suggestion
    public function getWebsiteStores(int $websiteId, bool $available = false): array
```